### PR TITLE
Updated and fixed AET In 10 Minutes wiki

### PR DESCRIPTION
**Pull request by mchrominski**
This is a change of the documentation page based on my experience with setting this up for the first time.

## Description
Minor changes and tweaks after having issues installing AET locally.

## Motivation and Context
This improves the overall initial experience for first touch with AET locally.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the [code style](https://github.com/Cognifide/aet/blob/master/CONTRIBUTING.md#coding-conventions) of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

---
I hereby agree to the terms of the AET Contributor License Agreement.

### CODE REVIEW

File: documentation/src/main/wiki/AETIn10Minutes.md [hunk 1]
```diff
@@ -8,19 +8,19 @@ Before start make sure that you have enough memory on your machine (8 GB is mini
...
 ### Set Up Vagrant
 
-Open a command prompt as the administrator and execute the following commands:
+Open a command prompt **as the administrator** and execute the following commands:
 * `vagrant plugin install vagrant-omnibus`
 * `vagrant plugin install vagrant-berkshelf`
 * `vagrant plugin install vagrant-hostmanager`
 
-Navigate to the `vagrant` module directory. Run `berks install` and then `vagrant up` to start the virtual machine. This process may take a few minutes.
+Clone the AET source code and navigate to the `vagrant` module directory. Run `berks install` and then `vagrant up` to start the virtual machine. This process may take a few minutes.
```
Comment (Skejven) [inline documentation/src/main/wiki/AETIn10Minutes.md]: Maybe good idea here is to provide the link or command?
```
git clone git@github.com:Cognifide/aet.git
```
There is also the alternative to cloning for people who don't have GitHub account - download zipped repo archive.

File: documentation/src/main/wiki/AETIn10Minutes.md [hunk 2]
```diff
@@ -8,19 +8,19 @@ Before start make sure that you have enough memory on your machine (8 GB is mini
 You need to download and install the following software:
    * [VirtualBox 5.1.30](https://www.virtualbox.org/wiki/Download_Old_Builds_5_1)
    * [Vagrant 1.9.2](https://releases.hashicorp.com/vagrant/)
-   * [ChefDK 1.1.16](https://downloads.chef.io/chef-dk/)
+   * [ChefDK 2.4.17](https://downloads.chef.io/chef-dk/)
```
Comment (Skejven) [inline documentation/src/main/wiki/AETIn10Minutes.md]: There is one more place in AET where this versions are listed:
https://github.com/Cognifide/aet/blob/master/vagrant/README.md#installation

Please update it also there.
Maybe even better option is to update `/vagrant/README.md` file to refer this wiki page? To keep all versions in one place.

File: documentation/src/main/wiki/AETIn10Minutes.md (hunk 1 above)
Comment (mchrominski) [reply documentation/src/main/wiki/AETIn10Minutes.md]: done

File: documentation/src/main/wiki/AETIn10Minutes.md (hunk 2 above)
Comment (mchrominski) [reply documentation/src/main/wiki/AETIn10Minutes.md]: I've simply updated the version to keep the changes minimal.

**Pull request merged**